docs/template_plugin/tests/functional/op_reference/base_reference_test.hpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ie_core.hpp>
+#include <ie_ngraph_utils.hpp>
+#include <ngraph/ngraph.hpp>
+#include <shared_test_classes/base/layer_test_utils.hpp>
+
+class CommonReferenceTest {
```

Comment:
I am not sure, but may be we will have some value from inheritance by 'LayerTestCommon()'? Some parts of the code look like duplication from this class

Reply:
`LayerTestCommon` is really huge class which contains SLT specific (like generation of input/output blobs), changing layouts and etc.
The idea was to use the separate class in order to avoid overcomplicated logic inside the `LayerTestCommon` class and make this class more developer friendly.